core/upgrade/upgrades.php

```diff
@@ -928,6 +929,14 @@ public static function _v_3_2_0_migrate_breakpoints_to_new_system( $updater, $in
 		return self::move_settings_to_kit( $callback, $updater, $include_revisions );
 	}
 
+	public static function _v_3_4_0_remove_old_elementor_scheme_() {
```

Comment:
```suggestion
	public static function _v_3_4_0_remove_old_elementor_scheme() {
```

Do you need this underscore?

Reply:
Yes, the same I did with another cleanup in another task. Because its removing a prefix.

Reply:
Handled.